components/logo/README.md

```diff
@@ -4,18 +4,67 @@
 
 ## Toegankelijkheid
 
-Soms is het logo de belangrijkste manier om te zien op welke website je bent, en wat de naam van de organisatie is. Zorg dan dat de tekst in het logo groot genoeg is en het contrast voldoende is, anders is de afzender niet duidelijk. WCAG maakt een uitzondering voor logo's, maar dat is geen goede reden om een onduidelijke site te hebben.
+Test je logo ook in _dark mode_ en in in _forced colors_ mode.
 
 ## Bestandsformaat
 
 Gebruik voor het logo bij voorkeur een een vectorafbeelding zoals SVG, zodat het logo scherp is op verschillende soorten schermen. Een pixel-logo (zoals PNG) is vaak onscherp op moderne schermen die een hoge pixeldichtheid hebben.
 
-Het is mogelijk een scherpe versie van een pixelafbeelding te hebben voor schermen met een hoge pixeldichtheid. Gebruik daarvoor [`srcset` met een onder andere een `2x` variant](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset). SVG gebruiken is meestal toch handiger, omdat je maar één bestand nodig hebt voor alle soorten pixeldichteid.
+Het is mogelijk een scherpe versie van een pixelafbeelding te hebben voor schermen met een hoge pixeldichtheid. Gebruik daarvoor [`srcset` met een onder andere een `2x` variant](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset). SVG gebruiken is meestal toch handiger, omdat je maar één bestand nodig hebt voor alle soorten pixeldichtheid.
 
 ## Text content
 
-Text embedded in the logo image will not be translated by tools built-in to browsers. Consider placing the logo text in the HTML document instead, so it is available for users reading a translation too.
+Als het logo tekst bevat, dan is het belangrijk voor gebruikers dat de letters voldoende groot zijn en de tekst voldoende contrast heeft. [WCAG eis 1.4.6](https://www.w3.org/TR/WCAG21/#contrast-enhanced) maakt voor logo's een uitzondering, maar het is niet gebruiksvriendelijk om hier een uitzondering te maken.
+
+Tekst in de afbeelding van het logo zal niet vertaald worden door automatische vertalingen van browsers. Overweeg of je de tekst uit de afbeelding in het HTML document kan plaatsen, zodat de tekst ook begrijpelijk is voor gebruikers van een automatische vertaling.
+
+## HTML
+
+### Tekst-alternatief
+
+Het moet duidelijk zijn dat de afbeelding een logo is. Als je de afbeelding ziet dan is het waarschijnlijk direct duidelijk. Als je de afbeelding niet ziet, dan weet je niet of de beschrijving "de gemeente Utrecht" gaat over een luchtfoto of een logo.
+
```

Comment:
Misschien kunnen we dit toevoegen:
- Beschrijf de afbeelding niet als “afbeelding" in de alternatieve tekst voor een afbeeldingslogo om redundantie te voorkomen (screenreader kondigt aan met welk soort elementen men interactie heeft op de pagina). Dus niet: “Afbeelding logo Gemeente Utrecht…” want dat wordt voorgelezen als “Afbeelding afbeelding…”
- Schermlezers lezen alle alt-tekst in je logo's voor, dus houd het beknopt zodat de gebruikerservaring er niet onder lijdt.